policyengine_us/variables/gov/states/hi/tax/income/subtractions/hi_military_pay_exclusion.py

```diff
@@ -11,3 +11,16 @@ class hi_military_pay_exclusion(Variable):
     )
     definition_period = YEAR
     defined_for = StateCode.HI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.hi.tax.income.subtractions.military_pay
+        person = tax_unit.members
+        military_service_income = person("military_service_income", period)
+        capped_military_income = min_(military_service_income, p.max_amount)
```

Comment:
is this capped per person or for the whole household?

Reply:
I think we should add the exclusion for both the head and the spouse here? 
![image](https://github.com/PolicyEngine/policyengine-us/assets/97819588/f906ffeb-3e02-4ddf-831b-0ec1865107fd)

Reply:
I think the way this is written is right, I believe it should be per person